### ISSUE TITLE
Read orbits from HORIZONS

### ISSUE THREAD

**Issue by Juanlu001** [poliastro/poliastro#313]
🐞 **Problem**

Although we can read asteroid data from two sources, we cannot read HORIZONS, arguably the most comprehensive online database on orbital data.

🎯 **Goal**

Be able to load an osculating orbit directly from HORIZONS, or even an array of positions.

💡 **Possible solutions**

This is the most popular Python library that reads from HORIZONS: https://github.com/mommermi/callhorizons

Notice that HORIZONS gives you the full history and not only an osculating orbit. We could choose to return both in some way, requires some API design.

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

**Comment by Juanlu001** [comment]
And for that, we should use https://astroquery.readthedocs.io/en/latest/jplhorizons/jplhorizons.html

**Comment by Juanlu001** [comment]
Something like this, but updated to use astroquery:

```
def get_orbit(name, epoch):
    # Get orbital data
    roadster = callhorizons.query(name, smallbody=False)
    roadster.set_discreteepochs([epoch.jd])
    roadster.get_elements(center="500@10")

    # Create Orbit object to do conversions
    roadster_eclip = Orbit.from_classical(
        Sun,
        roadster['a'][0] * u.au,
        roadster['e'][0] * u.one,
        roadster['incl'][0] * u.deg,
        roadster['node'][0] * u.deg,
        roadster['argper'][0] * u.deg,
        roadster['trueanomaly'][0] * u.deg,
        epoch
    )

    # Convert to ICRS
    roadster_eclip_coords = HeliocentricEclipticJ2000(
        x=roadster_eclip.r[0], y=roadster_eclip.r[1], z=roadster_eclip.r[2],
        v_x=roadster_eclip.v[0], v_y=roadster_eclip.v[1], v_z=roadster_eclip.v[2],
        representation=CartesianRepresentation,
        differential_type=CartesianDifferential,
        obstime=epoch
    )

    roadster_icrs_coords = roadster_eclip_coords.transform_to(ICRS)
    roadster_icrs_coords.representation = CartesianRepresentation

    # Create final orbit
    roadster_icrs = Orbit.from_vectors(
        Sun,
        r=[roadster_icrs_coords.x, roadster_icrs_coords.y, roadster_icrs_coords.z] * u.km,
        v=[roadster_icrs_coords.v_x, roadster_icrs_coords.v_y, roadster_icrs_coords.v_z] * (u.km / u.s),
        epoch=epoch
    )

    return roadster_icrs
```

**Comment by Juanlu001** [comment]
astroquery has a nasty bug at the moment: https://github.com/astropy/astroquery/issues/1192

**Comment by Juanlu001** [comment]
callhorizons stopped working as well: https://github.com/mommermi/callhorizons/issues/17

**Comment by Juanlu001** [comment]
I figured out how to properly use astroquery, so we already moved the notebook away from callhorizons.

**Comment by Juanlu001** [comment]
Just to clarify: astroquery is now the blessed way of accessing online astronomical data, including Horizons.

**Comment by Juanlu001** [comment]
I'm inclined towards adding a new `from_horizons` method to `poliastro.twobody.Orbit`.

**Comment by Juanlu001** [comment]
Closed in #519.
